### ISSUE TITLE
[Question] Is the default `action_noise` of off-policy algorithms, 0 noise

### ISSUE THREAD

**Issue by Kallinteris-Andreas** [DLR-RM/stable-baselines3#1547]
### ❓ Question

The default `action_noise` argument is `None`, of off-policy algorithms like TD3 and SAC, can you verify that this means that no policy/exploration noise is added, if not what is the default.

https://github.com/DLR-RM/stable-baselines3/blob/ffe26ccf95d7e3b37067bd81025d3b4b45825038/stable_baselines3/common/off_policy_algorithm.py#L377-L381

Thanks!

### Checklist

- [X] I have checked that there is no similar [issue](https://github.com/DLR-RM/stable-baselines3/issues) in the repo
- [X] I have read the [documentation](https://stable-baselines3.readthedocs.io/en/master/)
- [X] If code there is, it is minimal and working
- [X] If code there is, it is formatted using the [markdown code blocks](https://help.github.com/en/articles/creating-and-highlighting-code-blocks) for both code and stack traces.

**Comment by araffin** [comment]
> can you verify that this means that no policy/exploration noise is added, if not what is the default.

Yes, by default no additional action noise is added for SAC/TD3. But in the case of SAC, we the policy is stochastic, so there is already some noise that helps the exploration (https://proceedings.mlr.press/v164/raffin22a.html). In some cases (like mountain car), you can also add some external noise.

For TD3, it is more than recommended to add some action noise because the underlying policy is deterministic, that's why all our examples with DDPG/TD3 include external action noise: https://stable-baselines3.readthedocs.io/en/master/modules/td3.html#example
